easybuild/easyconfigs/s/scVelo/scVelo-0.3.1-foss-2023a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'scVelo'
+version = '0.3.1'
+local_commit = 'ffe2cd3'
```

Comment:
@pavelToman How did you determine this commit?

It doesn't correspond to the `0.3.1` version, see https://github.com/theislab/scvelo/releases

Reply:
I get this commit because it helps with `scvi-tools` and `scikit-learn` dependencies. 
Should I use `version = '0.3.1-20240314'` instead?

Reply:
Yes, it is about https://github.com/theislab/scvelo/pull/1185 + https://github.com/theislab/scvelo/pull/1194? They moved `scvi-tools` to optional-dependencies and unpin `scikit-learn < v1.2.0` restriction - there is a scikit-learn v1.3.1. for 2023a toolchain.
I am going to use 0.3.1 release and patch it, thank you for a clarification.